packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockListMenuContent.tsx

```diff
@@ -87,7 +87,7 @@ export const BlockListMenuContent: VFC<BlockListMenuContentProps> = ({
             blockSuggesterProps.onChange(option.variant, option.meta);
             popupState?.close();
           }}
-          key={`${option.meta.name}/${option.variant.name}`}
+          key={`${option.meta.componentId}/${option.variant.name}`}
```

Comment:
prevents key duplication if the same block is loaded from two different URLs (e.g. blockprotocol.org version, and local dev version)